File: 0344-reverse-string/0344-reverse-string.cs
public class Solution {
    public void ReverseString(char[] s) {
        for(int i = 0; i < s.Length/2; i++){
            //  https://hiperzstudio.tistory.com/32
            //  C# 7.0 이상부터 지원하는 튜플 형식
            (s[i], s[s.Length - 1 - i]) = (s[s.Length - 1 - i], s[i]);
        }
    }
}

File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int n = s.size();
        for(int i = 0; i < n/2; i++){
            char tmp = s[i];
            s[i] = s[n - 1 - i];
            s[n - 1 - i] = tmp;

            //  https://leetcode.com/problems/reverse-string/solutions/3275389/swapping-method-c-omar
            // swap(s[i], s[n-i-1]);
        }
    }
};